SwiftAppTemplate/SwiftAppTemplate/APIClient.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  APIClient.swift
+//  SwiftAppTemplate
+//
+//  Created by Catriona Scott on 5/1/19.
+//  Copyright © 2019 Catriona Scott. All rights reserved.
+//
+
+import Alamofire
+
+// TODO: Make this class implement the ConnectionTokenProvider protocol
+class APIClient {
+
+    /**
+     To get started with this demo, you'll need to first deploy an instance of
```

Comment:
Let's remove this comment, or make it more generic: "This example APIClient talks to an unauthenticated backend at this URL".